packages/app/src/modals-manager/reducers/modals-manager.interface.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import { PayloadAction } from "@reduxjs/toolkit"
+import { ModalsManagerEvent } from "App/modals-manager/constants"
+
+export enum ModalStateKey {
+  collectingDataModal = "collectingDataModalShow",
```

Comment:
I guess we should use capital letters in the enums

Reply:
https://github.com/mudita/mudita-center/pull/558/commits/9b3c44517019bac416e5873ca4d984aa6359e4b4